src/Renci.SshNet/Common/TimeSpanExtensions.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+#if NET21_OR_GREATER || NET6_0_OR_GREATER
```

Comment:
Do you mean `NETSTANDARD2_1_OR_GREATER`?

Reply:
Yes, you're right.

Reply:
According to [the docs](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.callerargumentexpressionattribute?view=net-8.0), the attribute became available in .NET Core 3.0, so I've updated it to `#if NETCOREAPP3_0_OR_GREATER`.